aws-sagemaker-hosted-scorer-cpp/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8
+
+RUN apt-get update
+
+RUN pip3 install https://s3.amazonaws.com/artifacts.h2o.ai/releases/ai/h2o/daimojo/2.7.5/x86_64-centos7/daimojo-2.7.5-cp38-cp38-linux_x86_64.whl
+
+RUN pip3 install datatable
+
+RUN pip3 install flask-restful
```

Comment:
You can install multiple pkgs right away.

Reply:
sure, I will modify docker config accordingly.